src/app/management/commands/delete_old_files.py

```diff
@@ -0,0 +1,27 @@
+import datetime
```

Comment:
Do we need this file now?

Reply:
Yes, actually for the testing I have made a django-admin command for that we are requiring this